pkg/mountpoint/args.go

```diff
@@ -94,6 +97,13 @@ func (a *Args) Set(key ArgKey, value ArgValue) {
 	a.args.Insert(arg{key, value})
 }
 
+// SetIfAbsent sets value of given key only if that key does not exist.
+func (a *Args) SetIfAbsent(key ArgKey, value ArgValue) {
+	if !a.Has(key) {
+		a.args.Insert(arg{key, value})
```

Comment:
```suggestion
		a.Set(key, value)
```

or

```suggestion
                key = normalizeKey(key)
		a.args.Insert(arg{key, value})
```

`Set` method already normalizes the key, if we don't use that we should normalize here as well